grove/Light_sensor/main.py

```diff
@@ -0,0 +1,28 @@
+from machine import ADC
+import zcoap
+import time
+import ujson
+
+def light_sensor():
+    ain1 = ADC(0)
+    raw = ain1.read()
+    v = (raw / 4096) * 0.6 * 6
```

Comment:
I understood 0.6 = "ADC internal reference" ,6 = "inverse num of default ADC gain 1/6" and 4096="ADC resolution is 12bit".  It may represent in code like below and it code more helpful.

> ADC_REF = 0.6
> ADC_RESOLUTION=4096 #12bit
> ain.gain(ain.GAIN_1_6) #gain set to 1/6
> v = (raw / ADC_RESOLUTION) * ADC_REF * 6

Reply:
I added a commit to fix it.

Reply:
Thank you.